include/envoy/upstream/load_balancer.h

```diff
@@ -76,6 +77,11 @@ class LoadBalancerContext {
    * Returns the set of socket options which should be applied on upstream connections
    */
   virtual Network::Socket::OptionsSharedPtr upstreamSocketOptions() const PURE;
+
+  /**
+   * Returns the transprot socket options which should be applied on upstream connections
```

Comment:
typo: transport

Reply:
Weird that the spell checker didn't find this.